src/hidl/impl-af.c

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2019 Jolla Ltd.
+ *
+ * Contact: Juho Hämäläinen <juho.hamalainen@jolla.com>
+ *
+ * This application is free software; you can redistribute
+ * it and/or modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ *
+ * This application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this application; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+ * USA.
+ */
+
+#include <gbinder.h>
+#include <glib-unix.h>
+
+#include "common.h"
+#include "impl.h"
+#include "logging.h"
+#include "dbus-comms.h"
+
+#define BINDER_DEVICE               GBINDER_DEFAULT_BINDER
+#define SERVICE_NAME                "media.audio_flinger"
```

Comment:
Does this replace the actual android media.audio_flinger?
If so then i think SERVICE_IFACE should be "android.media.IAudioFlinger".

I think it doesn't have an effect because this: https://github.com/mer-hybris/pulseaudio-modules-droid-hidl/pull/1/files#diff-6ce61f9a6d8e431cb63898c015ced1c4R72 doesn't check the interface, but @slava would know better what effects it has.

Reply:
You are correct and like you said, it doesn't have effect now. But I'll fix this.